ledger/error/src/main/scala/com/daml/error/definitions/IndexErrors.scala

```diff
@@ -8,6 +8,7 @@ import com.daml.error.definitions.ErrorGroups.ParticipantErrorGroup.IndexErrorGr
 import com.daml.error.utils.ErrorDetails
 import io.grpc.StatusRuntimeException
 
+@Explanation("This error group contains errors raised by the Participant Index persistence layer.")
```

Comment:
Since following error groups are introduced without much ado, I would be brief here as well
```suggestion
@Explanation("Errors raised by the Participant Index persistence layer.")
```